chainerx_cc/chainerx/python/array.cc

```diff
@@ -139,9 +139,16 @@ py::object MakeCupyArrayFromArray(const py::module& m, py::handle self) {
 
 ArrayBodyPtr MakeArray(py::handle object, py::handle dtype, bool copy, py::handle device) {
     absl::optional<Dtype> dtype_ = dtype.is_none() ? absl::nullopt : absl::optional<Dtype>(GetDtype(dtype));
-    Device& dev = GetDevice(device);
 
-    return MakeArray(object, dtype_, copy, dev);
+    Device* dev = nullptr;
```

Comment:
Please undo the fix to `MakeArray`.

Reply:
Like the blow `MakeArray` it shouldn't copy in some cases so we still need to check this.

Reply:
How about separating a PR? 
It's not used in this PR.

Reply:
Moved to https://github.com/chainer/chainer/pull/8296